common/schema-model/src/main/java/io/smallrye/graphql/schema/model/EnumValue.java

```diff
@@ -59,4 +59,9 @@ public void setDirectiveInstances(List<DirectiveInstance> directiveInstances) {
         this.directiveInstances = directiveInstances;
     }
 
+    public void addDirectiveInstance(DirectiveInstance directiveInstance) {
+        if (directiveInstances != null) {
```

Comment:
Probably not a real issue, but generally I'm not quite happy that this method will basically do **nothing** in case that `directiveInstances == null`. I'd rather see a NPE thrown than silently ignoring the call.

Reply:
Sure, I just went by the method `Field#addDirectiveInstances` https://github.com/smallrye/smallrye-graphql/blob/74107939776694ddc866b843d45f7a061f3cb3d8/common/schema-model/src/main/java/io/smallrye/graphql/schema/model/Field.java#L202
Should I also change the method `Field#addDirectiveInstance` which is right under it?